pom.xml

```diff
@@ -29,7 +29,7 @@
     <parent>
         <groupId>com.forgerock.sapi.gateway</groupId>
         <artifactId>secure-api-gateway-parent</artifactId>
-        <version>2.0.0</version>
+        <version>2.0.1-SNAPSHOT</version>
```

Comment:
Should we do a release of parent at this point - or are you still experimenting?

Reply:
We'll release it as part of the FatJoe release.